src/test/regress/sql/qp_correlated_query.sql

```diff
@@ -834,6 +834,28 @@ DROP TABLE skip_correlated_t4;
 reset optimizer_join_order;
 reset optimizer_trace_fallback;
 
+--------------------------------------------------------------------------------
+-- Ensure ORCA generates the correct plan with the exists clause
+-- for the partitioned table.
+--------------------------------------------------------------------------------
+CREATE TABLE offers (
+    id int,
+    product int,
+    date date
+) DISTRIBUTED BY (id);
+INSERT INTO offers SELECT 0, 0, '2023-01-01';
+CREATE TABLE contacts (
+    contact text,
+    id int,
+    date date
+) DISTRIBUTED BY (id) PARTITION BY RANGE(date) (START (date '2023-01-01') INCLUSIVE END (date '2023-02-01') EXCLUSIVE EVERY (INTERVAL '1 month'));
+INSERT INTO contacts SELECT '0', 0, '2023-01-01';
+EXPLAIN (COSTS off, VERBOSE on)
```

Comment:
This tests works even if second PcrFirst() is not replaced with PcrFirstUsed(). How can I test that replacing of second  PcrFirst() occurrence is correct?